tests/unit/classifier/GaussianProcessClassification_unittest.cc

```diff
@@ -490,6 +497,45 @@ TEST_F(GaussianProcessClassificationTest, get_mean_vector)
 	
 }
 
+TEST_F(GaussianProcessClassificationTest, train)
+{
+	std::shared_ptr<LikelihoodModel> likelihood1 =
+	    std::make_shared<SoftMaxLikelihood>();
+	likelihood1->put(random::kSeed, 1);
+	std::shared_ptr<Inference> inf =
+	    std::make_shared<MultiLaplaceInferenceMethod>();
+	std::shared_ptr<Kernel> k = std::make_shared<GaussianKernel>();
+	std::shared_ptr<MeanFunction> mean = std::make_shared<ZeroMean>();
+	inf->put("kernel", k);
+	inf->put("mean_function", mean);
+	inf->put("likelihood_model", likelihood1);
+	auto gpc1 = std::make_shared<GaussianProcessClassification>(inf);
+	gpc1->set_labels(multi_labels_train);
+	gpc1->train(features_train);
+	auto labels_predict1 = gpc1->apply_multiclass(features_test);
+	auto labels1 = labels_predict1->get<SGVector<float64_t>>("labels");
+
+	std::shared_ptr<LikelihoodModel> likelihood2 =
+	    std::make_shared<SoftMaxLikelihood>();
+	std::shared_ptr<Inference> inf2 =
+	    std::make_shared<MultiLaplaceInferenceMethod>();
+	inf2->put("kernel", k);
+	inf2->put("mean_function", mean);
+	inf2->put("likelihood_model", likelihood2);
+	auto gpc2 = std::make_shared<GaussianProcessClassification>(inf2);
+	gpc2->set_labels(multi_labels_train);
+	gpc2->put("seed", 1);
+	gpc2->train(features_train);
+	auto labels_predict2 = gpc2->apply_multiclass(features_test);
+	auto labels2 = labels_predict2->get<SGVector<float64_t>>("labels");
+
+	for (int i = 0; i < 25; i++)
+	{
+		EXPECT_NEAR(
+		    labels1[1], labels2[1], std::numeric_limits<float64_t>::epsilon());
```

Comment:
`i`? You are just checking 25 times the same thing here no?

Reply:
sorry, yes, it should be `i`.